### DISCUSSION TITLE
Add ask password protal

### DISCUSSION THREAD

**Discussion by lnee94** [flatpak/xdg-desktop-portal#1273] [New Portals]
So there is a concern that a window could intercept a password box so lets create where app asks a portal to make a password box and pkexec style would add info like what app this is coming from. This window would have prority over other windows to display over all.

**Comment by TingPing** [comment]
I'm not sure I understand the risk here. If an application asks for your password of course it will get the password?

**Comment by ebassi** [comment]
No response/clarification after a year and a half.
